openpype/settings/defaults/project_settings/maya.json

```
@@ -204,7 +204,8 @@
         "ValidateFrameRange": {
             "enabled": true,
             "optional": true,
-            "active": true
+            "active": true,
+            "exclude_families": []
```

I would move default families from `validate_frame_range.py#30` to here.
```suggestion
            "exclude_families": ["model", "rig", "staticMesh"]
```